tests/cart_shift_1.f90

```diff
@@ -0,0 +1,31 @@
+program cart_shift_1
```

Comment:
this test file doesn't work the same way with and without our MPI wrappers, there is a difference.